_config.yml

```diff
@@ -756,6 +756,14 @@ twikoo:
   region: ap-shanghai
   path: window.location.pathname
 
+# Cusdis
+# 基于第三方服务或自托管服务
+# Based on third-party or self-hosted service
+# See https://cusdis.com
+cusdis:
+  host:
+  appID:
+
```

Comment:
Could add a `lang` to support i18n: https://cusdis.com/doc#/advanced/i18n

Reply:
Support i18n and open a PR to merge into dev branch.